src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java

```diff
@@ -48,6 +48,10 @@ public FoodMeeting(int id, String name, String imageLink, Timestamp eventDate, U
 		this(id, name, imageLink, FoodMeetingStatus.Voting, eventDate, eventDate, eventDate, eventDate, null, userOwner);
 	}
 
+	public FoodMeeting(int id, String name, String imageLink, FoodMeetingStatus status, Timestamp eventDate, User userOwner) {
+		this(id, name, imageLink, status, eventDate, eventDate, eventDate, eventDate, null, userOwner);
```

Comment:
If I understand this, here we are creating the FoodMeetings without payment time. Is that correct? if so, is there a reason to do that?